### PR TITLE
Update block-merge branch to final LFS version

### PR DESCRIPTION
**Pull request by nzpr**
Merging latest work in dev and LFS, same branch as in https://github.com/rchain/rchain/pull/3215 used

### CODE REVIEW

File: rholang/src/test/scala/coop/rchain/rholang/InterpreterSpec.scala [hunk 1]
```diff
@@ -85,26 +85,23 @@ class InterpreterSpec extends FlatSpec with Matchers {
             """.stripMargin
               )
 
-          contents <- storageContents(runtime)
-        } yield contents
+          tupleSpace <- runtime.space.toMap
```
Comment (zsluedem) [inline rholang/src/test/scala/coop/rchain/rholang/InterpreterSpec.scala]: use ` tupleSpace <-runtime.getHotChanges`

Comment (nzpr) [reply rholang/src/test/scala/coop/rchain/rholang/InterpreterSpec.scala]: Thanks!

**Review by zsluedem — APPROVED** [review]
LGTM

**Pull request merged**